packages/cog/src/cog/cutline.ts

```diff
@@ -8,6 +8,12 @@ import { CoveringPercentage, CutlineZoom, ZoomDifferenceForMaxImage } from './co
 import { CogJob, SourceMetadata } from './types';
 
 const PaddingFactor = 1.125;
+type CutlineItem = { path: string; blend: number };
+export const CutlineMap: { [key: string]: CutlineItem } = {
+    sentinel: { path: 's3://basemaps-cog-test/NZCoastCutline.sentinel.geojson', blend: 5 },
```

Comment:
We cannot have the full path to these stored here as they vary based on prod vs dev.

maybe just leave it as the key `urban` `sentinel` `rural` then we can use the key in the `output` location possibly `s3://${output}/cutline/${cutline}.json`

Reply:
I wondered about that. Wasn't sure where to get the bucket name. Will do

Reply:
As above. Used outputConfig.path